output/output.go

```diff
@@ -15,7 +15,7 @@ import (
 // Output is interface for output
 type Output interface {
 	OutputSchema(wr io.Writer, s *schema.Schema) error
-	OutputTable(wr io.Writer, s *schema.Table) error
+	OutputTable(wr io.Writer, s *schema.Table, viewpoints schema.Viewpoints) error
```

Comment:
@k1LoW 
To refer viewpoints from tables, I need to chenge the `Output` interface and that affects other unrelated files like `dot.go`.
If you have better idea to implement this feature, kindly let me know.
I appletiate it.

Reply:
It may be better to add a [Viewpoints](https://github.com/k1LoW/tbls/blob/5059fd987a596def1d1c350da019745a0a84d6fe/schema/schema.go#L72) field to the [Table](https://github.com/k1LoW/tbls/blob/5059fd987a596def1d1c350da019745a0a84d6fe/schema/schema.go#L135-L147).

Reply:
@macoto1995 You may have it implemented as is. I will correct it on my end!

Reply:
@k1LoW 
Oh..Sorry I missed your new comment for a few days.

> It may be better to add a [Viewpoints](https://github.com/k1LoW/tbls/blob/5059fd987a596def1d1c350da019745a0a84d6fe/schema/schema.go#L72) field to the [Table](https://github.com/k1LoW/tbls/blob/5059fd987a596def1d1c350da019745a0a84d6fe/schema/schema.go#L135-L147).
I have some concern about this solution. 
Since `Viewpoint` has `Tables` as children, it would be infinite loop.
Plus, to render link to viewpoints from the markdown, the `OutputTable` function has to know the index of the viewpoints, which is not included in `Viewpoint` type.

So I suppose that it's better to have `TableViewpoint` type like below.

```go
struct TableViewpoint {
    Index Int
    Name String
    Desc String
}
```

Kindly, let me know what you think.
I'd work on this way for now. 
Thank you.

Reply:
> Since Viewpoint has Tables as children, it would be infinite loop.

Yes. In fact, recursive relationships are everywhere (e.g. `Table.ReferencedTables` and `Table.Column.ParentRelations` ).

> Plus, to render link to viewpoints from the markdown, the OutputTable function has to know the index of the viewpoints, which is not included in Viewpoint type.

fmfm, indeed. Good point.

> So I suppose that it's better to have TableViewpoint type like below.

LGTM. So far I have no better ideas either.

Reply:
Thanks! I'll work on this way!